subworkflows/local/variant_calling/call_snv_sentieon.nf

```diff
@@ -33,11 +35,17 @@ workflow CALL_SNV_SENTIEON {
             ch_vcf_idx = ch_vcf.join( ch_index )
 
             SENTIEON_DNAMODELAPPLY ( ch_vcf_idx, fasta, fai, ml_model )
-            ch_vcf      = SENTIEON_DNAMODELAPPLY.out.vcf
-            ch_index    = SENTIEON_DNAMODELAPPLY.out.vcf_index
             ch_versions = ch_versions.mix(SENTIEON_DNAMODELAPPLY.out.versions.first())
-        }
 
+            BCF_FILTER_ONE ( SENTIEON_DNAMODELAPPLY.out.vcf )
+
+            BCF_FILTER_TWO ( BCF_FILTER_ONE.out.vcf )
```

Comment:
It is possible that you want to have `BCF_FILTER_TWO` and the `TABIX_BCFTOOLS` processes after this if statement. Depends on whether the vcf from the `DNASCOPE` process has added filters to variants and you want those removed.

Reply:
Oh, alright  - you mean if `BCF_FILTER_TWO` should filter output from `DNASCOPE`? We will check!

Reply:
@jemten  I've checked the output of dnascope and it's kind of lacking filters. At least nothing with 'Pass' or 'Fail'. So if we run the bcftools_filter_tow (given the argument: -i FILTER="PASS" ) on the raw output from dnascope, it would leave us nothing. I think it should be applied on the vcf right after the first filtering.

Reply:
But the ML_PROB info that bcftools_filter_one uses exists in the dnascope output as well.  So we can have the filtering outside o the if block. @jemten What do you think?

Reply:
Hmm, have I understood this correctly?
_Dnacope and dnamodelapply both add/modify the INFO/FILTER tag ML_PROB on each variant. You only want to keep the variants passing the filters._ 
In that case both the bcftools_filter processes should be placed after the if statement.